src/fabric/src/fabric.erl

```diff
@@ -503,26 +504,30 @@ cleanup_index_files() ->
 %% @doc clean up index files for a specific db
 -spec cleanup_index_files(dbname()) -> ok.
 cleanup_index_files(DbName) ->
+    lists:foreach(fn(File) ->
```

Comment:
Getting a syntax error here:
`/Users/jay/proj/couchdb/src/fabric/src/fabric.erl:507: syntax error before: '->'`
One of the downsides of mixing (heh) elixir and erlang?
s/fn/fun/